sys/net/gnrc/netif/ethernet/gnrc_netif_ethernet.c

```diff
@@ -196,6 +198,12 @@ static gnrc_pktsnip_t *_recv(gnrc_netif_t *netif)
             gnrc_pktbuf_realloc_data(pkt, nread);
         }
 
```

Comment:
Why not put `addr_str[]` here?

Reply:
A larger array like this I rather put into global space as it is done through-out the rest of the code base.